source/content/guides/wordpress-configurations/05-wordpress-cache-plugin.md

```diff
@@ -39,12 +39,22 @@ You can increase the default time to live value to improve the chances that a vi
 
 1. Modify the **Default Cache Time**.
 
-    You should strike a balance between freshness of content and speed. We recommend a minimum of 600 seconds. If you can increase the setting to 30 minutes (1800 seconds) or 1 hour (3600 seconds), many more requests will hit the Edge Cache. Every page served from the Edge Cache won't hit your application container's PHP workers or MySQL database, which means faster page load times and a better user experience for site visitors.
+    When [Pantheon Advanced Page Cache](https://wordpress.org/plugins/pantheon-advanced-page-cache) is active, your site content should always be fresh and served quickly from Pantheon's Edge Cache. This is why we recommend caching content for a minimum of 1 week. Every page served from the Edge Cache won't hit your application container's PHP workers or MySQL database, which means faster page load times and a better user experience for site visitors.
 
 1. Click **Save Changes**.
 
 ![WordPress Pantheon Cache Plugin settings](../../../images/WordPress_Pantheon-Cache-Settings.png)
 
+### Override the default TTL
+
```

Comment:
we should use `max-age`, not `ttl` in this filter

Reply:
Or should we? https://github.com/pantheon-systems/documentation/pull/8953#pullrequestreview-2020329072